src/components/footer/Footer.js

```diff
@@ -15,9 +16,9 @@ export default function Footer() {
           About us
         </a>
         {/* TODO link the link below to the header */}
-        <a href="#header" className="footer__top transparency">
+        <Link to="header" className="footer__top transparency">
```

Comment:
Sweet! The issue mentioned has been noted [here](https://github.com/McCambley/apiary/issues/25).